contents/blog/product-health-metrics.md

```diff
@@ -0,0 +1,127 @@
+
+- signups are continuing
+- you are keeping users engaged
+- users aren’t leaving
+
+Active users are the lifeblood of your product, so seeing them continue to grow means a product is healthy. Blood must continue to flow for your body to work, the same is true with your product.
+
+Doing a ratio of these values such as DAU/MAU can also be useful. This helps you understand how frequently people are engaging with the product. You can use PostHog’s formula mode to calculate this. 
+
+![DAU/MAU ratio](../images/blog/product-health-metrics/daumau.png)
+
+### Feature usage
+
+Having users is great, but you want them to actually use your product. Measuring feature usage helps you do this. This is tracking the usage of parts of your product, such as creating, analyzing, or sharing content.
+
+Feature usage helps you understand the health of individual parts of your product. Like the body, if you heal the "sick" parts, the whole is healthier. Feature usage improves prioritization by aligning development time with usage.
+
+Feature usage is a more specific health metric than new user growth or churn rate. It shows the health of specific features, which enables it to be more actionable.
+
+### Session duration
+
+Depending on the product, session duration, or its average, are useful product health metrics. It is a general measurement of user engagement. This is because it shows if users are spending significant time in your product. If this was to change significantly, something fundamentally shifted in your app.
+
+For some products, the extremes of session duration might be more important. This enables you to monitor the health of your product for power users, which could be different, but more important, than average users.
+
+![Session duration](../images/blog/product-health-metrics/duration.png)
+
+## How to choose the right product health metrics
+
+The "right" product health metric depends on your product, company, stage, industry, and infinitely more variables. Picking one goes back to what makes a good product health metric. It must be something you want to:
+
+- improve slowly, but a dramatic change would cause concern and intervention.
+- monitor in the short run (daily, weekly) and improve in the long run.
+- know and track because it makes a difference in the quality of the product and experience for users.
+
+Choosing the right level of detail is also important. Being too specific causes the metric to not be representative of the product’s health. Being too broad means the metric isn’t actionable and makes it unclear what interventions you can do to improve an issue.
+
+Connecting health metrics to OKRs helps make them specific and actionable. It also creates ownership that helps the product improve. Improving the product, and users’ experience with it, is what health metrics should ideally do. If they aren’t, like a diet that doesn’t see results, it’s a sign to rethink your health regime towards something that helps your product improve.
```

Comment:
I think what this section needs is some examples based on product types – i.e. here's what good could look like for a B2C xyz product, for an online marketplace, B2B SaaS and so on.

Reply:
Added some, let me know what you think.